script/Arcueid/Day 4/04_00_ARC04_6_2D.txt

```diff
@@ -38,7 +38,7 @@ How on earth did you kill me?"
 -- Page 565, Offset 7091.
 -- ルーンやカバラあたりの秘術には抗体が出来ているから効かないし、抗体耐性が出来てない<神|も><秘|の>―――わたしがまだ経験した事のない魔術となると、この国の古神道と南米の秘宝ぐらいなものよ」
 My antibodies can render secret arts like Runes and Kabbalah useless,
- but they couldn't mount any resistance whatsoever against whatever %{i}Arcanum%{/i} you used.
+ but they couldn't mount any resistance whatsoever against whatever arcanum you used.
  If we're talking about magecraft I haven't experienced before,
  it would have to be either the ancient Shintoism of this country or the hidden treasures of South America."
```

Comment:
runes and kabbalah probably don't need capitalization

secret arts > mysticism